shared/src/test/scala/mouse/AnyFSyntaxTest.scala

```diff
@@ -6,19 +6,36 @@ import cats.{Id, ~>}
 
 class AnyFSyntaxTest extends MouseSuite {
 
-  val emptyK: List ~> List = λ[List ~> List](_ => Nil)
+  val emptyK = new (List ~> List) {
+    def apply[A](x: List[A]) = Nil
+  }
```

Comment:
This syntax is not supported in Dotty, so I opted to write it like this. It's only in tests, so it seemed fair to me.